vector/src/main/java/im/vector/activity/CountryPickerActivity.java

```diff
@@ -72,6 +73,7 @@ public static Intent getIntent(final Context context, final boolean withIndicato
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        ThemeUtils.activitySetTheme(this);
```

Comment:
I can remove all of these repeated `activitySetTheme` calls if I make a common superclass to all Vector activities, does that sound like a good idea to you?